ddl/000013_export_repair.down.sql

```diff
@@ -0,0 +1,3 @@
+update catalog_branches_export_state
+set state='exported-successfully'
+where state = 'export-repaired'
```

Comment:
1. Please use caps for SQL keywords.
2. Please end with a semicolon (so it is safe to add additional commands) and a newline (so editors like it).

Also on the next SQL file.

Reply:
Changed, Thanks